src/components/Spinner.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import styled, { keyframes } from 'styled-components';
+
+/**
+ * Loading animation from:
+ * https://loading.io/css/
+ *
+ * And implemented using style-components animations:
+ * https://styled-components.com/docs/basics#animations
+ */
+
+const Animation = keyframes`
+  0% {
+    top: 8px;
+    height: 64px;
+  }
+  50%, 100% {
+    top: 24px;
+    height: 32px;
+  }
+`
```

Comment:
similar question goes for css data. Can this be imported efficiently instead of packagin in the code itself?

Reply:
So... sort of.... react you can render the CSS in a few different ways. 

`<file>.css` that's just the traditional way. You use the class, and id and whatever else to render the css. That file can become pretty big making it hard to navigate if you have a lot of styles. 

`sytle = {}` those are basically injecting the style directly into the `style=` property of the tag.

`styled-components` so that's the style here. It basically creates a sudo tag `<Spinner />` with a defined style. The advantage is that you can now reuse very simple components with defined CSS that you don't have to hunt down. Disavantage is that it fragments the CSS. 

Each approach has pros and cons... We can go with full traditional `file.css`. I'm mostly trying this package and seeing how it changes the way I approach UX design.